src/common/transform_json.py

```diff
@@ -0,0 +1,214 @@
+import glob
+import boto3
+import os
+import json
+import re
+from datetime import datetime
+from src.common.sagemaker_config_sample import CRDCDH_S3_BUCKET, RAW_DATA_PREFIX, TRAIN_DATA_PREFIX
```

Comment:
Need create sagemaker_config.py by copying the sagemaker_config_sample.py and updated if needed.  Then change the line 7 to 
from common.sagemaker_config import CRDCDH_S3_BUCKET, RAW_DATA_PREFIX, TRAIN_DATA_PREFIX